buttonconfig.cpp

```diff
@@ -19,6 +21,15 @@ buttonConfig::~buttonConfig()
 
 void buttonConfig::on_buttonBox_accepted()
 {
-    qDebug() << ui->lineEdit_1->text();
     emit(configAccepted(ui->lineEdit_1->text(), ui->lineEdit_2->text(), ui->lineEdit_3->text(), ui->lineEdit_4->text(), ui->lineEdit_5->text(), ui->lineEdit_6->text()));
 }
+
+void buttonConfig::receiveButtons(QString one, QString two, QString three, QString four, QString five, QString six)
```

Comment:
This looks like you can only change the buttons at runtime, and they will be reset next time the program is launched.

You may want to checkout the QSettings thing to store settings and load settings at launch: http://doc.qt.io/qt-5/qsettings.html

Reply:
That's somewhere on my roadmap, yeah.